packages/types/src/primitive/Method.ts

```diff
@@ -128,7 +128,7 @@ export default class Method extends Struct implements IMethod {
       args: new Uint8Array(),
       argsDef: {},
       meta: new FunctionMetadata(),
-      callIndex: new Uint8Array([255, 255])
+      callIndex: new Uint8Array()
```

Comment:
Throw on line 125, line 88 becomes -

`private static decodeMethod (value: DecodedMethod | Uint8Array | string = new Uint8Array(), _meta?: FunctionMetadata): DecodedMethod {`

Reply:
I changed that (if I understood you correctly)

Reply:
100%, line 125 should throw, everything below that should be removed. (No return - at that point, we give up)